learning_journal/learning_journal/views/default.py

```diff
@@ -14,11 +14,8 @@
 )
 def list_view(request):
     """List of journal entries."""
-    return {
-        'title': 'Main',
-        'entries': ENTRIES
-    }
-
+    entries = request.dbsession.query(Entry).all()
+    return {'entries': entries}
 
 @view_config(
     route_name='detail_view',
```

Comment:
Expand your views to meet spec. We need an edit view and a notfound view. You should also remove the corpse code in your default.py.